packages/shared/src/styles/base.css

```diff
@@ -153,6 +154,11 @@ body {
   --theme-color-water: theme('colors.water.40');
   --theme-color-salt: theme('colors.salt.40');
   --theme-color-email: theme('colors.pepper.10');
+
+  --theme-squads-avocado: theme('colors.avocado.10');
+  --theme-squads-bacon: theme('colors.bacon.60');
+  --theme-squads-cabbage: theme('colors.cabbage.20');
+  --theme-squads-onion: theme('colors.onion.90');
```

Comment:
Should we not have a setup for light mode too? Or are they the same?

Reply:
They are the same, exclusive colors that mark specific squads.
So watercooler is always the same color (it's more an accent)

Reply:
Yeah they stay the same as these colours based on an image which won't change in light/dark mode

Reply:
Amazing. Thank you both for clearing it up.